README.md

```diff
@@ -103,6 +103,9 @@ An implementation is not compliant if it fails to satisfy one or more of the MUS
 
   <dt>Secret</dt>
   <dd>A Kubernetes <a href="https://kubernetes.io/docs/concepts/configuration/secret/"><code>Secret</code></a></dd>
+
+  <dt><code>PodSpec</code>-able resource</dt>
```

Comment:
```suggestion
  <dt><code>PodSpec</code>-able Duck Type</dt>
```

Reply:
@nebhale This is changed after your approval. Please check again.